resources/js/src/app/components/basket/AddItemToBasketOverlay.js

```diff
@@ -129,12 +129,24 @@ Vue.component("add-item-to-basket-overlay", {
                 return "";
             }
 
-            const orderParam = orderParams.find(param =>
+            const property = this.latestBasketEntry.item.properties.find(property =>
             {
-                return parseInt(param.property.id) === parseInt(propertyId);
+                return parseInt(property.property.id) === parseInt(propertyId);
             });
 
-            return orderParam.property.value;
+            if (isNullOrUndefined(property) || !property.property.isOderProperty)
```

Comment:
Die Komponente AddToBasket und SingleItem verwenden nur properties die den Wert "property.isShownOnItemPage" true besitzen, muss die Prüfung hier auch rein?

Reply:
Gute Frage. Was passiert mit (verpflichtenden) OrderProperties, die nicht sichtbar sein sollen

Reply:
Die bisherige Logik sieht OrderProperties nur als Pflicht vor, wenn diese auch als sichtbar eingestellt ist.

Den entsprechenden Code findet man hier: 
[SingleItemModule.js](https://github.com/plentymarkets/plugin-ceres/blob/5f9c0f6554c10be29d86e19ab99ecd59e0122266/resources/js/src/app/store/modules/SingleItemModule.js#L169)

und hier:

[AddToBasket.js](https://github.com/plentymarkets/plugin-ceres/blob/5f9c0f6554c10be29d86e19ab99ecd59e0122266/resources/js/src/app/components/basket/AddToBasket.js#L99)

Reply:
Eine Filterung an dieser Stelle nach "property.isShownOnItemPage" ist nicht nötig, da in diesem Fall das Merkmal gar nicht zum Warenkorb hinzugefügt wird und somit gar nicht erst in den orderParams auftaucht.